docs/guides/android_chiptool_building.md

```diff
@@ -62,18 +62,24 @@ Complete the following steps to build CHIPTool:
 
 1. Check out the CHIP repository.
 
-2. In the command line, run the following command from the top CHIP directory:
+2. Run bootstrap (**only required first time**)
```

Comment:
next time requires `source scripts/activate.sh` (runs faster, but needed).

Did we consider replacing running android_app.sh with

```
./scripts/build/build_examples.py --platform android build
```

or maybe 

```
./scripts/build/build_examples.py --platform android --board arm64 build
```

?

Reply:
Currently `android_app.sh` runs `source scripts/activate.sh` for you, so that's why I didn't include that step.

I'm in favor of switching to `build_examples.py`, but didn't want to change anything yet as I haven't taken the time to personally test that script yet. I could do it later in a separate PR (maybe in one where we remove android_app.sh entirely).

Reply:
That is fair. I also see #8875 adding cmakelist support but still preserving 'the old way' and I imagine at some  point we want to move back to having a single way of compiling things (one script, one method).